content/learn/book/welcome/apps/_index.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+Bevy programs store and execute all of their game logic and data with a single {{rust_type(type="struct" crate="bevy" mod = "app" name="App" no_mod = "true")}} data structure.
+
+Let's make a trivial Hello World app to demonstrate how that works in practice.
+The process is straightforward: we first create a new {{rust_type(type="struct" crate="bevy" mod = "app" name="App" no_mod = "true")}}.
+Then, we add a simple system, which prints "Hello, Bevy!" when it is run.
+Finally once we're done configuring the app, we call {{rust_type(type="struct" crate="bevy" mod = "app" name="App" method="run" no_mod = "true")}} to actually make our app *do things*.
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+    App::new()
+        .add_system(hello)
+        .run();
+}
+
+fn hello(){
+    println!("Hello, Bevy!")
+}
+```
+
+## What makes an App?
+
+So, what sort of data does our  {{rust_type(type="struct" crate="bevy" mod = "app" name="App" no_mod = "true")}} really store?
+Looking at the docs linked, we find three fields: `world`, `schedule` and `runner`.
+The `world` field stores all of our game's data, the `schedule` holds the systems that operate on this data (and the order in which they do so) and the `runner` interprets the schedule to control the broad execution strategy.
+You can read more about these by exploring the reference documentation linked just above.
```

Comment:
It is not immediately obvious to me which documentation link this is referencing.